ui-admin/src/study/families/FamilyLink.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Family } from '@juniper/ui-core'
+import { Link } from 'react-router-dom'
+import { StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import { getFamilyNames } from 'util/familyUtils'
+import { isEmpty } from 'lodash'
+
+/**
+ * Links to a family, similar to the EnrolleeLink component.
+ */
+export const FamilyLink = ({
+  studyEnvContext,
+  family
+}: {
+  studyEnvContext: StudyEnvContextT,
+  family: Family
+}) => {
+  const path = `${studyEnvContext.currentEnvPath}/families/${family.shortcode}`
+  const familyNames = getFamilyNames(family)
```

Comment:
To make it clearer that this isn't an array
```suggestion
  const familyNameString = getFamilyNameString(family)
```